plugins/module_utils/core.py

```diff
@@ -375,6 +376,8 @@ def scrub_none_parameters(parameters):
     for k, v in parameters.items():
         if isinstance(v, dict):
             clean_parameters[k] = scrub_none_parameters(v)
```

Comment:
```suggestion
            clean_parameters[k] = scrub_none_parameters(v, descend_into_lists=descend_into_lists)
```
Shouldn't it passed on recursively? In any case, the documenation should be extended, because it doesn't mention recursion *at all*.